---
title: "From GWAS to signal validation: A novel method for estimating SNP genetic effects while preserving genomic context"

# Authors
# If you created a profile for a user (e.g. the default `admin` user), write the username (folder name) here 
# and it will be replaced with their full name and linked to their profile.
authors:
- Scott Wolf 
- Varada Abhyankar
- admin
- Julien F. Ayroles
- Luisa F. Pallares

date: "2023-03-13T00:00:00Z"
doi: ""

# Schedule page publish date (NOT publication's date).
publishDate: "2023-03-08T00:00:00Z"

# Publication type.
# Legend: 0 = Uncategorized; 1 = Conference paper; 2 = Journal article;
# 3 = Preprint / Working Paper; 4 = Report; 5 = Book; 6 = Book section;
# 7 = Thesis; 8 = Patent
publication_types: ["3"]

# Publication name and optional abbreviated publication name.
publication: bioRxiv

abstract:  "Validating associations between genotypic and phenotypic variation remains a challenge, despite advancements in association studies. Common approaches for signal validation rely on gene-level perturbations, such as loss-of-function mutations or RNAi, which can only identify effects coarsely. More recent CRISPR-based methods can validate associations at the SNP level, but have significant drawbacks, including resulting off-target effects and being both time-consuming and expensive. Both approaches usually modify the genome of a single genetic background, limiting the generalizability of experiments. To address these challenges, we present a simple, low-cost experimental scheme for validating genetic associations at the SNP level in outbred populations. The approach involves genotyping live outbred individuals at a focal SNP, crossing homozygous individuals with the same genotype at that locus, and contrasting phenotypes across resulting synthetic outbred populations. We tested this method in Drosophila melanogaster, measuring the phenotypic effects of a polymorphism at a naturally-segregating cis-eQTL for the midway gene. Our results demonstrate this method's utility in SNP-level validations of naturally occurring genetic variation regulating complex traits. This method provides a bridge between statistical discovery and validation in the natural context of outbred populations, offering a potential solution to the limitations of current methods."
tags: [gene expression, phenotypic variation,  RNAseq, Gene Ontology, transcriptional variance, gene expression variance ]

# Display this page in the Featured widget?
featured: true

url_pdf: 'publication/FocalSNP/Wolf_etal_2023.pdf'
url_code: ''
url_dataset: ''
url_poster: ''
url_project: ''
url_slides: ''
url_source: ''
url_video: ''

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
image:
  caption: ''
  focal_point: ""
  preview_only: false

---

